experimental/babel-preset-env/package.json

```diff
@@ -57,7 +57,7 @@
     "@babel/core": "7.0.0-beta.31",
     "@babel/helper-fixtures": "7.0.0-beta.31",
     "@babel/helper-plugin-test-runner": "7.0.0-beta.31",
-    "compat-table": "kangax/compat-table#957f1ff15972e8fb2892a172f985e9af27bf1c75",
+    "compat-table": "https://api.github.com/repos/kangax/compat-table/tarball/957f1ff15972e8fb2892a172f985e9af27bf1c75",
```

Comment:
I'm wondering why you changed this. Did you encounter an issue? I don't think we should use the API directly since we will likely be rate limited.

Reply:
This was a yarn-related bug, which I think may actually be fixed in the latest version.

Reply:
Alright, I've reverted this change and rebased to latest master

Reply:
Thanks.